invenio_github/config.py

```diff
@@ -55,8 +57,35 @@
 GITHUB_RELEASE_CLASS = 'invenio_github.api:GitHubRelease'
 """GitHubRelease class to be used for release handling."""
 
-GITHUB_METADATA_FILE = '.zenodo.json'
+GITHUB_METADATA_FILE = '.invenio.json'
 """File with extra metadata stored in GitHub repository."""
 
 GITHUB_DEPOSIT_CLASS = 'invenio_deposit.api:Deposit'
 """Deposit class that implements a `publish` method."""
+
+GITHUB_PID_FETCHER = 'recid'
+"""PID Fetcher for Release records."""
+
+GITHUB_REMOTE_APP = dict(
```

Comment:
What about getting a deep copy of the default value from invenio_oauthclient.contrib.cern and updating only needed keys?